cpp/src/parquet/arrow/arrow-reader-writer-test.cc

```diff
@@ -441,6 +504,16 @@ static std::shared_ptr<GroupNode> MakeSimpleSchema(const ::DataType& type,
   return std::static_pointer_cast<GroupNode>(node_);
 }
 
+std::shared_ptr<::arrow::Schema> ToArrowSchema(const GroupNode& schema_node) {
```

Comment:
Is this https://github.com/apache/arrow/blob/master/cpp/src/parquet/arrow/schema.cc#L336?

Reply:
Thanks, that's handy. I'll refactor to use that